src/mgmt/srv.c

```diff
@@ -1099,6 +1099,54 @@ ds_mgmt_profile_hdlr(crt_rpc_t *rpc)
 	crt_reply_send(rpc);
 }
 
+/**
+ * Set mark on all of server targets
+ */
+void
+ds_mgmt_mark_hdlr(crt_rpc_t *rpc)
+{
+	struct mgmt_mark_in	*in;
+	crt_opcode_t		opc;
+	int			topo;
+	crt_rpc_t		*tc_req;
+	struct mgmt_mark_in	*tc_in;
+	struct mgmt_mark_out	*out;
+	int			rc;
+
+	in = crt_req_get(rpc);
+	D_ASSERT(in != NULL);
+
+	topo = crt_tree_topo(CRT_TREE_KNOMIAL, 32);
+	opc = DAOS_RPC_OPCODE(MGMT_TGT_MARK, DAOS_MGMT_MODULE,
+			      DAOS_MGMT_VERSION);
+	rc = crt_corpc_req_create(dss_get_module_info()->dmi_ctx, NULL, NULL,
+				  opc, NULL, NULL, 0, topo, &tc_req);
```

Comment:
You guys are aware that using this in-band approach we will suffer from killed ranks, right?

Reply:
yes, but this is the easy way to ship the function at the moment. And given the use case, is probably ok for now. though we probably can retrieve the killed ranks from management server and skip them in future.